client/src/containers/BillingPage.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { Elements, StripeProvider } from 'react-stripe-elements';
+import CheckoutForm from './CheckoutForm';
+
+const BillingPage = props => {
+  const [subType, setSubType] = useState('');
+
+  const handleChange = e => {
+    setSubType(e.target.value);
+  };
+
+  return (
+    <>
+      <h1>BillingPage</h1>
+      <StripeProvider apiKey="pk_test_6uEhds8mHz26DG95ZvUwTURp">
```

Comment:
At some point you are going to want to create an environment variable for this, but since it appears to be a test key you shouldn't get any $30,000 charges or anything :)